build/images/ovs/apply-patches.sh

```diff
@@ -67,4 +67,8 @@ if version_get "$OVS_VERSION" "2.13.0"; then
     # but this seems like an acceptable workaround.
     sed -i 's/python3\.7/python3\.6/' debian/openvswitch-test.install
     sed -i 's/python3\.7/python3\.6/' debian/python3-openvswitch.install
+
+    # OVS ovs-monitor-ipsec is Python2 code but others have upgraded to Python3. Before upstream updates,
+    # here ovs-monitor-ipsec should be modified.
```

Comment:
suggest this comment instead:
The OVS ovs-monitor-ipsec script has a Python3 shebang but still includes some Python2-specific code. Until the patch which fixes the script is merged upstream, we apply it here, or Antrea IPsec support will be broken.